shell/platform/common/accessibility_bridge.cc

```diff
@@ -95,6 +96,14 @@ void AccessibilityBridge::CommitUpdates() {
     }
   }
 
+  // Set the tree's root if necessary. The new root is the last list's first
+  // node.
+  if (!results.empty() && GetRootAsAXNode()->id() == ui::AXNode::kInvalidAXID) {
```

Comment:
what if the root node changes? or would it never change?

Reply:
A view's root node never changes once it is created. I can tweak this comment to make this more clear

Reply:
I tweaked this comment, let me know what you think!